pages/index.tsx

```diff
@@ -30,18 +41,68 @@ export default () => {
       }),
   })
 
+  const { payload: user, isLoading: isLoadingUser } = query<UserData>({
+    queryKey: ['baseUser', session?.token],
+    queryFn: () =>
+      axios({
+        url: '/api/v3/user',
+        method: GET,
+        headers: { Authorization: `Bearer ${session?.token}` },
+      }),
+    enabled: loggedIn,
+  })
+
+  const { payload: pendingTrades, isLoading: isLoadingPendingTrades } = query<
+    ListResponse<TradeCard>
+  >({
+    queryKey: ['pending-trades', String(user?.uid)],
+    queryFn: () =>
+      axios({
+        method: 'GET',
+        url: `/api/v3/trades?username=${user?.uid}&status=PENDING`,
+      }),
+    enabled: !!user?.uid,
+  })
+
   const limitedCards = useMemo(
     () => (packs?.length ? packs.slice(0, 25) : []),
     [packs]
   )
-
+  console.log(isLoadingUser)
   return (
     <PageWrapper>
       <div className="space-y-8">
-        <h1 className="text-3xl font-bold text-center mb-4">
-          Welcome to the Ice Level
-        </h1>
-        <div className="mb-4">
+        {isLoadingPendingTrades ? (
```

Comment:
This could be simplified to

```
<h1 className="text-3xl font-bold text-center mb-4">
  Welcome to Ice Level {user && `${user.username}`}
</h1>
{user && pendingTrades?.total > 0 && (
  <Alert className="text-black text-xl" status="info">
    <AlertIcon />
      <Link href={`/trade`}>
      {' '}
      Welcome back {user.username}, you have{' '}
      {pendingTrades.total} pending trades{' '}
      </Link>
  </Alert>
)}
)}
```